cs/explore/ExplorationStrategies.cs

```diff
@@ -0,0 +1,152 @@
+using System;
```

Comment:
Structurally, I was imagining grouping this by functionality (exploration) rather than by language (cs).  Should we make this a subdirectory of /explore instead?

Reply:
Happy to move it, my only concern would be that we would get an odd duplication of hierarchies, since we would probably want to put a /cs/ folder under /explore?

Reply:
explore/cs seems preferable to cs/explore since it will help us keep these in sync.  Furthermore, I expect we will refer people to the directory based on functionality more than language.

Although git lacks links, perhaps leaving a small file at cs/explore pointing to explore/cs is a reasonable way of handling this?